File: ReadMLB.Web.Scout/src/app/players/player-running/runningStat.model.ts
export class RunningStatModel {
  public year: number;
  public league: number;
  public teamId: number;
  public teamName: string;
  public inPO: boolean;
  public rs: number;
  public sb: number;
  public cs: number;
}

export class DefenseStatModel {
  public year: number;
  public league: number;
  public teamId: number;
  public teamName: string;
  public inPO: boolean;
  public asst: number;
  public po: number;
  public err: number;
  public fld: number;
}

export class FieldRunningStatModel {
  public year: number;
  public league: number;
  public teamId: number;
  public teamName: string;
  public inPO: boolean;
  public asst: number;
  public po: number;
  public err: number;
  public fld: number;
  public rs: number;
  public sb: number;
  public cs: number;
}

File: ReadMLB.Web.API/Model/RunningStatModel.cs

namespace ReadMLB.Web.API.Model
{
    public class RunningStatModel
    {
        public long RunningId { get; set; }
        public short Year { get; set; }
        public byte TeamId { get; set; }

        public string TeamAbr { get; set; }
        public string TeamName { get; set; }

        public string Organization { get; set; }
        public byte League { get; set; }
        public bool InPO { get; set; }
        public short RS { get; set; }
        public short SB { get; set; }
        public short CS { get; set; }
    }
}
